crates/bevy_math/src/rotation2d.rs

```diff
@@ -331,10 +331,20 @@ impl Rot2 {
 
     /// Returns the angle in radians needed to make `self` and `other` coincide.
     #[inline]
+    #[deprecated(
+        since = "0.15.0",
+        note = "Use `angle_to` instead, the semantics of `angle_between` will change in the future."
+    )]
     pub fn angle_between(self, other: Self) -> f32 {
         (other * self.inverse()).as_radians()
```

Comment:
Nit: This could now use `angle_to`. This'll be removed later though so it doesn't really matter

```suggestion
        self.angle_to(other)
```

Reply:
Ah, dangit I spotted this locally but forgot to push